packages/zowe-explorer/src/commands/TsoCommandHandler.ts

```diff
@@ -301,7 +182,7 @@ export class TsoCommandHandler extends ZoweCommandProvider {
             };
             tsoParms.account = await Gui.showInputBox(InputBoxOptions);
             if (!tsoParms.account) {
-                Gui.showMessage(vscode.l10n.t("Operation Cancelled."));
+                Gui.showMessage(this.operationCancelled);
```

Comment:
Thoughts on removing this "operation cancelled" message? 😋

Reply:
I think we should, since we have seen a decrease of them in other PRs 🙏 
Will resolve the comment once I commit the changes 🙏

Reply:
resolved by 7f1375eb5e2250a29a5a98773c699f405e9a77be